src/runtime/cli/kata-check.go

```diff
@@ -416,6 +417,11 @@ EXAMPLES:
 			return errors.New("check: cannot determine runtime config")
 		}
 
+		// check if cgroup can work use the same logic for creating containers
+		if _, err := vc.V1Constraints(); err != nil && err == cgroups.ErrMountPointNotExist && !runtimeConfig.SandboxCgroupOnly {
+			return fmt.Errorf("Cgroup v2 only works with SandboxCgroupOnly set to true. Please read https://github.com/kata-containers/kata-containers/blob/main/docs/design/host-cgroups.md#cgroups-v2 for details.")
```

Comment:
```suggestion
			return fmt.Errorf("Cgroup v2 requires the following conifguration: `sandbox_cgroup_only=true`.")
```

Reply:
Hi @fidencio, have updated it as your suggestion.